zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -652,20 +658,15 @@ impl<F: Field> BytecodeCircuitConfig<F> {
     fn set_padding_row(
         &self,
         region: &mut Region<'_, F>,
-        challenges: &Challenges<Value<F>>,
         offset: usize,
         last_row_offset: usize,
     ) -> Result<(), Error> {
-        let empty_hash = challenges
-            .evm_word()
-            .map(|challenge| rlc::value(EMPTY_CODE_HASH_LE.as_ref(), challenge));
-
         self.set_row(
             region,
             BytecodeCircuitRow {
                 offset,
                 last_row_offset,
-                code_hash: empty_hash,
+                code_hash: empty_code_hash_word_value(),
```

Comment:
I suppose we are using big-endian here because the output of `keccak([])` is in big-endian?

Reply:
I think `From<eth_types::Hash> for Word<F>` implementation reverses the endianness, right?

Reply:
Oh, I see. So we are doing little endian here.
Can we doc the endianness in empty_code_hash_word_value?

Reply:
Done